contractcourt/htlc_success_resolver.go

```diff
@@ -201,50 +238,168 @@ func (h *htlcSuccessResolver) Resolve() (ContractResolver, error) {
 			&sweepTXID,
 			channeldb.ResolverOutcomeClaimed,
 		)
-	}
 
-	log.Infof("%T(%x): broadcasting second-layer transition tx: %v",
-		h, h.htlc.RHash[:], spew.Sdump(h.htlcResolution.SignedSuccessTx))
+	// If we have non-nil SignDetails, this means that have a 2nd level
```

Comment:
With this new statement, can move the two cases into distinct methods to allow them to be unit testable in isolation, and also cut down on some of the vertical space here.

Reply:
Done!